code/test/CoreTemplateStudio.Core.Test/CoreTemplateStudio.Core.Test.csproj

```diff
@@ -95,6 +90,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="AutoFixture" Version="4.17.0" />
```

Comment:
This seems not to be used, we should remove it. Reviewing nuget packages I also saw that Microsoft.NET.Test.Sdk has an update we missed before.

Reply:
true !! I had a look but didn´t use it in the end!